STX.REST.RESTFulSense.Clients.Tests.Unit/Services/Foundations/HttpExchanges/HttpExchangeServiceTests.Headers.cs

```diff
@@ -0,0 +1,1645 @@
+                InvalidHttpExchangeHeaderException = invalidHttpExchangeHeaderException
+            };
+        }
+
+        private static dynamic CreateAllowHeaderException(string[] invalidAllowHeader)
+        {
+            var invalidHttpExchangeHeaderException = new InvalidHttpExchangeHeaderException(
+              message: "Invalid HttpExchange content header error occurred, fix errors and try again.");
+
+            invalidHttpExchangeHeaderException.UpsertDataList(
+                key: nameof(HttpExchangeContentHeaders.Allow),
+                value: "Allow header has invalid configuration, fix errors and try again.");
+
+            HttpExchangeContentHeaders httpExchangeContentHeaders = new HttpExchangeContentHeaders
+            {
+                Allow = invalidAllowHeader
+            };
+
+            return new
+            {
+                HttpExchangeContentHeaders = httpExchangeContentHeaders,
+                InvalidHttpExchangeHeaderException = invalidHttpExchangeHeaderException
+            };
+        }
+
+        private static dynamic CreateContentDispositionHeaderException(
+            ContentDispositionHeader invalidContentDispositionHeader)
+        {
+            var invalidHttpExchangeHeaderException = new InvalidHttpExchangeHeaderException(
+              message: "Invalid HttpExchange request header error occurred, fix errors and try again.");
+
+            invalidHttpExchangeHeaderException.UpsertDataList(
+                key: nameof(HttpExchangeContentHeaders.ContentDisposition),
+                value: "Content Disposition header has invalid configuration, fix errors and try again.");
+
+            HttpExchangeContentHeaders httpExchangeContentHeaders = new HttpExchangeContentHeaders
+            {
+                ContentDisposition = invalidContentDispositionHeader
+            };
+
+            return new
+            {
+                HttpExchangeContentHeaders = httpExchangeContentHeaders,
+                InvalidHttpExchangeHeaderException = invalidHttpExchangeHeaderException
+            };
+        }
+
+        private static dynamic CreateContentEncodingHeaderException(string[] invalidContentEncodingHeader)
+        {
+            var invalidHttpExchangeHeaderException = new InvalidHttpExchangeHeaderException(
+              message: "Invalid HttpExchange content header error occurred, fix errors and try again.");
+
+            invalidHttpExchangeHeaderException.UpsertDataList(
+                key: nameof(HttpExchangeContentHeaders.Allow),
+                value: "Content Encoding header has invalid configuration, fix errors and try again.");
+
+            HttpExchangeContentHeaders httpExchangeRequestHeaders = new HttpExchangeContentHeaders
+            {
+                ContentEncoding = invalidContentEncodingHeader
+            };
+
+            return new
+            {
+                HttpExchangeRequestHeaders = httpExchangeRequestHeaders,
```

Comment:
Invalid Property (left side) and invalid variable name (right side)

Reply:
Resolved